.gitignore

```diff
@@ -35,9 +35,6 @@ vignettes/*.pdf
 # Shiny token, see https://shiny.rstudio.com/articles/shinyapps.html
 rsconnect/
 
-# targets
-_targets/
```

Comment:
Wouldn't this mean targets objects get uploaded to github including data files?

Reply:
There's another .gitignore in the _targets folder that excludes everything except itself and the meta file. That comes baked in with the default targets setup, for exactly this reason.